src/Button/Button.js

```diff
@@ -26,6 +26,8 @@ type Props = {
   circular?: boolean,
   /** Disables the Button */
   disabled?: boolean,
+  /** Element to be used as the root node - e.g. 'a' */
+  element?: string,
```

Comment:
_Edge case:_ Disabled buttons using an `element` of `a` do not prevent the link from being navigated.  Prolly not an issue since normal links can not really be "disabled".

Reply:
What do you think we should do in this case?